documentation/src/main/asciidoc/stories/assembly_rest_api.adoc

```diff
@@ -3,6 +3,7 @@
 = Interacting with the {brandname} REST API
 The {brandname} REST API lets you monitor, maintain, and manage {brandname}
 deployments and provides access to your data.
+If the response code from the service is not explicitly defined, `200 (OK)` means success.
```

Comment:
Hey, @oraNod 
It could be possible, but we need to say that some services can return `204` as success, but some `204`. Don't have an idea. We should describe a default service behavior.

Reply:
@dmvolod Cool. How about we just make this a note like:

[NOTE]
====
By default {brandname} REST API operations return `200 (OK)` when successful.

However, when some operations are processed successfully, they return an HTTP status code such as `204` or `202` instead of `200`.
====

Reply:
Excellent, thanks @oraNod 
Will update soon